flipperdevices/flipperzero-firmware
# [FL-1050] SD card error view

# What's new

- Add error check in sd card api
- Check error from file select

# Verification

- Build for f4 and f5 targets
- Try load saved ibutton keys when SD card is removed. Verify correct error notification

# Checklist (do not modify)
